src/lib/Components/Artist/ArtistArtworks/ArtistNotableWorksRail.tsx

```diff
@@ -50,7 +50,7 @@ const ArtistNotableWorksRail: React.FC<ArtistNotableWorksRailProps> = ({ artist
 
   return (
     <Box>
-      <Box mt={2} mb={1}>
+      <Box mt={2} mb={0}>
```

Comment:
Can mb just be left out here?
```suggestion
      <Box mt={2}>
```
Nitpicky/nonblocking